active/gridsyntax/GridSyntaxSpec2.md

```diff
@@ -0,0 +1,368 @@
+
+
+# Background
+_This adds a new short-hand Xaml syntax feature that makes it much easier to define a Grid in Xaml makrup. The proposal for this feature can be found [here.](https://github.com/microsoft/microsoft-ui-xaml/issues/673) The existing API documentation for Grid can be found [here.](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.grid)_
```

Comment:
It recommend to use the trademark (official name) XAML - in upper case, instead of Xaml. Xaml is correct when you refers to a namespace. This is the same case of .NET, it's .NET neither .Net nor .net.

Reply:
I don't think anything in here refers to the XAML product, it's all the Xaml language, which I believe follow the API casing guidelines (PascalCasing).